docs/wpf/dialog-service.md

```diff
@@ -107,6 +107,7 @@ To register a dialog, you must have a View (UserControl) and a corresponding Vie
  protected override void RegisterTypes(IContainerRegistry containerRegistry)
  {
      containerRegistry.RegisterDialog<NotificationDialog, NotificationDialogViewModel>();
+     //or simply: containerRegistry.RegisterDialog<NotificationDialog>(); if you are using AutoWireViewModel=True
```

Comment:
This isn't the difference between registering the Dialog or the Dialog & ViewModel. The difference is that we register the ViewModel with the ViewModelLocationProvider ahead of time so that it avoids the reflection lookup. It's the reflection avoidance which is why we tend to suggest registering the pair together and have stopped advocating simply registering the View by itself.

Reply:
Hi Dan, that's valuable information as well. for me that's fine as I just learnt it. would you like another suggestion to have that in the documentation or do we leave it as is and just close this issue?